rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -274,16 +233,7 @@ class Publisher : public PublisherBase
     auto ptr = MessageAllocTraits::allocate(*message_allocator_.get(), 1);
     MessageAllocTraits::construct(*message_allocator_.get(), ptr, msg);
     MessageUniquePtr unique_msg(ptr, message_deleter_);
-    return this->publish(unique_msg);
-  }
-
-  virtual void
-  publish(const MessageT * msg)
-  {
-    if (!msg) {
-      throw std::runtime_error("msg argument is nullptr");
-    }
-    return this->publish(*msg);
+    return this->publish(std::move(unique_msg));
```

Comment:
Looks like CI wants another include for `std::move`

http://build.ros2.org/job/Cpr__rclcpp__ubuntu_bionic_amd64/195/testReport/junit/rclcpp/cpplint/build_include_what_you_use__4____tmp_ws_src_rclcpp_rclcpp_include_rclcpp_publisher_hpp_236_/

```
Add #include <utility> for move
```